Dockerfile-alpine

```diff
@@ -0,0 +1,13 @@
+FROM golang:latest as builder
+
+RUN go get github.com/asobti/kube-monkey
+WORKDIR $GOPATH/src/github.com/asobti/kube-monkey
+#RUN make build
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o kube-monkey
+
+FROM alpine
+LABEL maintainer="asobti <ayushsobti@gmail.com>"
+
+RUN	apk update && apk upgrade && apk add --no-cache bash tzdata
```

Comment:
alphine doesn't have golang.
```
Sending build context to Docker daemon  150.1MB
Step 1/3 : FROM alpine
 ---> 11cd0b38bc3c
Step 2/3 : LABEL maintainer="asobti <ayushsobti@gmail.com>"
 ---> Running in d407f4128551
Removing intermediate container d407f4128551
 ---> 94627fdd335d
Step 3/3 : RUN  apk update && apk upgrade && apk add --no-cache bash tzdata
 ---> Running in f3d70013509b
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
v3.8.0-9-g716ebb36f3 [http://dl-cdn.alpinelinux.org/alpine/v3.8/main]
v3.8.0-9-g716ebb36f3 [http://dl-cdn.alpinelinux.org/alpine/v3.8/community]
OK: 9532 distinct packages available
OK: 4 MiB in 13 packages
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
(1/6) Installing ncurses-terminfo-base (6.1-r0)
(2/6) Installing ncurses-terminfo (6.1-r0)
(3/6) Installing ncurses-libs (6.1-r0)
(4/6) Installing readline (7.0.003-r0)
(5/6) Installing bash (4.4.19-r1)
Executing bash-4.4.19-r1.post-install
(6/6) Installing tzdata (2018d-r1)
Executing busybox-1.28.4-r0.trigger
OK: 16 MiB in 19 packages
Removing intermediate container f3d70013509b
 ---> 5e1b31a11202
...
$ docker run -it kube-monkey:test sh                                                                                                                        
/ # go
sh: go: not found
```

Reply:
In order to make docker image small Alpine do not have golang. 

As per my understanding, we are 1st building Kube-monkey from golang docker. We are copying executable in Alpine, which is a general practice that's why GO is not installed on Alpine.

Reply:
My bad, I assumed go required go to run the executable like java.